design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -104,24 +99,15 @@ type DeleteNegotiableQuotesOutput {
     # Implementation Note: We don't make the deleted quotes accessible
     # because "deleted" means they're hidden from the storefront UI. They will
     # still be visible (and labeled as deleted) for a Seller in the admin
-    negotiable_quotes(
-        filter: NegotiableQuoteFilterInput,
-        pageSize: Int = 20,
-        currentPage: Int = 1
-    ): NegotiableQuotesOutput @doc(description: "List of negotiable quotes available to customer")
+    quote: Boolean
 }
 
 input CloseNegotiableQuotesInput {
     quote_ids: [ID!]! @doc(description: "A List of IDs from NegotiableQuote objects")
 }
 
 type CloseNegotiableQuotesOutput {
-    closed_quotes: [NegotiableQuote]! @doc(description: "Quotes that were just closed")
-    negotiable_quotes(
```

Comment:
Why were these fields removed? `closed_quotes` and `negotiable_quotes`?

Reply:
@DrewML I have only removed negotiable_quotes from the output. Keep closed_quotes as it is. I am not sure can we use filtration on output also? The negotiable_quotes field has to add a user input. How can we set filter on output?
'''type CloseNegotiableQuotesOutput {
}'''
is the Output of closeNegotiableQuote.

Reply:
Here an example (it will not work because user is not logged in after creation, but shows how you can do filtration on the output). Let me know if I misunderstood the question.
```
mutation {
  createCustomer(input: {email: "email@domain.com", firstname: "John", lastname: "Doe", password: "test123$"}) {
    customer {
      email
      orders (filter: {number: {eq: "1"}}) {
        items {
          id
        }
      }
    }
  }
}
```

Reply:
@melnikovi Thanks I am looking for that filterable in the output. I have that query only. thanks. I need to look core example of how we can filter out the result on the output.